docs/reference-guides/block-api/block-transforms.md

```diff
@@ -373,6 +373,52 @@ transforms: {
 
 {% end %}
 
+<h4 id="schemas-and-content-models">Schemas and Content Models</h4>
+
+When pasting content it's possible to define a [content model](https://html.spec.whatwg.org/multipage/dom.html#content-models) that will be used to validate and process pasted content.
+It's often the case that HTML pasted into the editor will contain a mixutre of elements that _should_ transfer as well as elements that _shouldn't_.
```

Comment:
```suggestion
It's often the case that HTML pasted into the editor will contain a mixture of elements that _should_ transfer as well as elements that _shouldn't_.
```

Reply:
thanks for the catch!

Reply:
> I'm not familiar at all with how this works technically. From my view this is a helpful explanation of this concept, but I can't verify that it is technically correct or isn't missing something critical 😅

That's the point of this PR 😄

I've attempted to explain what the code actually does. It could be wrong, but I have tried to be accurate here and only explain things I confirmed.

Reply:
Makes sense! I'm hoping someone with more familiarity can confirm there aren't any nuances that should be included. From a "know nothing" perspective like my own this is a huge improvement over the non-existent previous documentation 😁